conf/variant/common/local.conf

```diff
@@ -20,3 +20,7 @@ DIRFILES_pn-nativesdk-cmake = "1"
 
 # Don't install any extra devel packages just because we're running on QEMU
 POKYQEMUDEPS_remove = "packagegroup-core-device-devel"
+
+# Don't build .hddimg and .iso image formats as we don't need it.
+NOHDD = "1"
+NOISO = "1"
```

Comment:
Can we use the more override-friendly version ?= to assign NOHDD and NOISO here?

Reply:
I guess it can still be overridden in a machine specific local.conf if it's done after including common/local.conf, so I guess it's ok to keep the assigment as it is now.